### PR TITLE
upgrade ng-annotate to a maintained fork

### PR DESCRIPTION
**Pull request by vinzd**
As a side note, this commit fixes our angular builds.

### CODE REVIEW

File: js/webpack_get_defaults.js [hunk 1]
```diff
@@ -121,6 +121,18 @@ function getDevtool () {
...
+function getBabelPlugins () {
+  const plugins = [
+    'transform-strict-mode',
+    ['transform-runtime', {polyfill: false}],
+    'transform-class-properties'
+  ]
+
+  if (config.build.type === enums.buildTypes.APPLICATION && config.build.profile === 'angular') {
+    plugins.push('angularjs-annotate')
+	}
```
Comment (vperron) [inline js/webpack_get_defaults.js]: warning : tabs

File: package.json [hunk 2]
```diff
@@ -39,6 +39,7 @@
     "babel-core": "6.24.0",
     "babel-eslint": "7.2.1",
     "babel-loader": "6.4.1",
+    "babel-plugin-angularjs-annotate": "0.1.0",
```
Comment (vperron) [inline package.json]: supprime l'ancien aussi.

File: js/webpack_get_defaults.js (hunk 1 above)
Comment (vinzd) [inline js/webpack_get_defaults.js]: en effet, merci

File: package.json (hunk 2 above)
Comment (vinzd) [inline package.json]: déjà fait, plus bas :)

**Pull request merged**